Formula/unit.rb

```diff
@@ -0,0 +1,95 @@
+class Unit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+  head "https://hg.nginx.org/unit", :using => :hg
```

Comment:
Remove head, which we only use for well-established formulas with large user base.